content/blog/check_my_repo.md

```diff
@@ -0,0 +1,150 @@
+---
+title: (Ͼ˳Ͽ)...check-my-repo!!!
+date: 2022-01-14T09:00:00+02:00
+description: This is a text about open source, and everything you need to be proud of your repository. Because open standards ensure not only the quality and shareability of a project with necessary documentation to make it welcoming, understandable and keeping an ongoing clear communication. Most importantly, open standards guide technology development by gently enforcing space for diversity and equity.
+author: 'Paloma Oliveira'
+authorTwitterHandle: 'pcultural'
+images:
+    - /images/blog/check_my_repo_steps.png
+tags:
+    - open source
+    - open governance
+    - standards
+    - diversity and equity
+---
+
+This is a text about open source, and everything you need to be proud of your repository. Because open standards ensure not only the quality and shareability of a project with necessary documentation to make it welcoming, understandable and keeping an ongoing clear communication. Most importantly, open standards guide technology development by gently enforcing space for diversity and equity.
+
+If you share my admiration for standards, optimistic that this common agreement is capable of making the world a little bit better; if you want to improve your messy project no one understands to make it more shareable; or if you just want to show off how cool you are because you have the most compliant repositories, this text is for you.
+
+## Open source as an ecosystem
+
+For some, open source is only a technical definition about code source that is open to copy, modify and distribute. Others may defend that, before you can even participate in an open source project, you must first learn to collaborate. That&#39;s because open source is not a verb nor one thing, it&#39;s an approach to software development, a workflow, a philosophy. Whatever your understanding may be, it is surely more than having a code publicly available.
+
+My personal definition is that open source is an ecosystem composed of technical things like code, but mostly of people. Just placing code on a public repository will not achieve the ideal of making it shareable, as mostly, being open means open to the &quot;other&quot; which is not me or clones of me. And this &quot;other&quot; should be able to be part of the project. But… how to include this other when you have no idea who they may be? This is where standards come into play.
```

Comment:
Oinnnn this is so lovely!!! Can I quote you? If so, I would add this in between both paragraphs... and this will be the 1st chaoter of my PhD o_O

> As a matter of fact, even the Open Source Initiative, one of the most active organizations since the 90's in the education and promotion of non-proprietary code, although focused on a definition particularly concerned with legal licensing, includes aspects beyond the technical, such as "No Discrimination Against Persons or Groups". 
> Christopher Hiller, recognized developer within the JavaScript community, author of two books on the language and respected maintainer of key projects like Node and Mocha, mentions the importance of including humans behind the code by design. 
> When you share your code, you (should) have some idea about who is going to use it. What does this audience expect? What are the norms in this particular community?
> The "other" lies at the edge of this audience, I think. It's not that we've shared software for everybody and anybody, but rather the people who would be interested in contributing to the software should necessarily have some common ground. And how you idealize the project's community will very much have an impact on the the diversity of contributors. If you focus entirely on "the code", your project may meet the OSI definition of open source, but you will likely have a homogenous contributor base.

Reply:
haha, cool.  maybe mention me as a fellow SL employee, and replace Mocha with Appium ;)